dumb-init.c

```diff
@@ -28,40 +28,18 @@ pid_t child_pid = -1;
 char debug = 0;
 char use_setsid = 1;
 
-void signal_handler(int signum) {
-    DEBUG("Received signal %d.\n", signum);
-
+void forward_signal(int signum) {
```

Comment:
separated out `forward_signal` and `handle_signal` to avoid confusing debug messages (otherwise it would print "Received signal $TERM" every time dumb-init exited, even though possibly nothing received TERM).